tcollector.py

```diff
@@ -682,6 +688,33 @@ def send_data(self):
         # the packets out of the kernel's queue
 
 
+    def send_data_to_signalfx(self):
+        try:
+            gauges = []
+            for line in self.sendq:
+                split_line = self.add_tags_to_line(line).split()
```

Comment:
The add_tags_to_line method modifies `line` in place. Is this something we could call more than once?